api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/Constants.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.dhsc.htbhf.claimant.model;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public final class Constants {
+
+    // Using simplified regex due to limitations of java unicode in regular expressions.
```

Comment:
The regex is much simpler than the version used in the frontend due to difference in regex behaviour between JS and Java. The key thing is that all emails that are valid in the UI are also valid here.